nipoppy/workflow/dicom_org/check_dicom_status.py

```diff
@@ -126,8 +127,13 @@ def run(global_config_file, regenerate=False, empty=False):
                 'See sample_dicom_dir_func.py for an example.'
             )
 
-        df_status[COL_PARTICIPANT_DICOM_DIR] = df_status[COL_SUBJECT_MANIFEST].apply(
-            lambda participant_id: participant_id_to_dicom_dir(participant_id, global_config)
+        df_status[COL_PARTICIPANT_DICOM_DIR] = df_status.apply(
+            lambda row: participant_id_to_dicom_dir(
```

Comment:
Cool, didn't know you could just pass in a row!

Reply:
Yes, turns out `apply` is not only for `Series`: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.apply.html